tests/container/Containerfile

```diff
@@ -11,6 +11,7 @@ RUN yum install \
     python-tox \
     python3-samba \
     python3-wheel \
+    python3-pyxattr \
```

Comment:
yep. I want to test some of this manually too, but it's easier to add the needed stuff to samba-containers after the code is merged in sambacc (otherwise you *have* to have a draft/wip pr in order to avoid accidentally using unmerged sambacc branches/commits).

I also am thinking we may want to add the ability to build sambacc as an RPM instead of a wheel. For the rpm based distros we use in samba-container this could avoid having to repeat ourselves wrt dependencies.